clouddriver-eureka/src/main/groovy/com/netflix/spinnaker/clouddriver/eureka/model/EurekaInstance.groovy

```diff
@@ -97,7 +99,8 @@ class DiscoveryInstance implements Health {
       vipAddress,
       secureVipAddress,
       lastUpdatedTimestamp,
-      asgName)
+      asgName,
+      metadata?.titusTaskId)
```

Comment:
goes away when we use instanceId for titus but worth considering datacenterinfo.metadata